packages/render-html/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-native-render-html",
-  "version": "6.3.4",
+  "version": "6.4.0",
```

Comment:
I noticed that the version number was wrong. So, we could merge this PR and try deploying with the new workflow since version 6.4.0 was never published on npm.

Reply:
Technically, it is correct; it shows the currently released version to NPM. The version 6.4.0 release on GitHub should be removed, taking a look.

Reply:
I removed the misleading 6.4.0 release from GitHub.